packages/babel-plugin-transform-classes/test/fixtures/loose/method-name-non-bmp-babel-7/output.js

```diff
@@ -0,0 +1,13 @@
+var o = /*#__PURE__*/function () {
+  "use strict";
+
+  function o() {
+    babelHelpers.classCallCheck(this, o);
+  }
+
+  babelHelpers.createClass(o, [{
+    key: "\uD842\uDFB7\u91CE\u5BB6",
+    value: function 𠮷野家() {}
```

Comment:
In current main it is transformed to
```js
  babelHelpers.createClass(o, [{
    key: "\uD842\uDFB7\u91CE\u5BB6"
  }]
```
here the `value` is dropped because `nameFunction` returns `undefined`.